easybuild/easyconfigs/e/Exporter-Lite/Exporter-Lite-0.08-GCCcore-7.3.0-Perl-5.28.0.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'PerlModule'
+
+name = 'Exporter-Lite'
+version = '0.08'
+
+homepage = 'https://metacpan.org/release/Exporter-Lite'
+description = "Exporter::Lite - lightweight exporting of functions and variables"
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+
+source_urls = ['https://cpan.metacpan.org/authors/id/N/NE/NEILB']
+sources = [SOURCE_TAR_GZ]
+checksums = ['c05b3909af4cb86f36495e94a599d23ebab42be7a18efd0d141fc1586309dac2']
+
+perl = 'Perl'
+perlver = '5.28.0'
```

Comment:
don't we have a `%(perlver)s`?

Reply:
Should've checked, we do have %(perlver)s
fixing...